src/styles/default/pretty.jl

```diff
@@ -283,27 +283,26 @@ end
 
 function format_docstring(style::AbstractStyle, state::State, text::AbstractString)
     state_indent = state.indent
-    boundaries = findall(text) do character
-        character != '"'
-    end
+    start_boundary = findfirst(!=('"'), text)
+    end_boundary = findlast(!=('"'), text)
     # first, we need to remove any user indent
     # only some lines will "count" towards increasing the user indent
     # start at a very big guess
     user_indent = typemax(Int)
-    user_indented = text[boundaries[1]:boundaries[end]]
+    user_indented = text[start_boundary:end_boundary]
     deindented = IOBuffer()
     user_lines = split(user_indented, '\n')
     for (index, line) in enumerate(user_lines)
         # the first line doesn't count
         if index != 1
-            first_character = findfirst(character -> !isspace(character), line)
```

Comment:
This returns a valid index of the string `line` (or `nothing`), but generally `first_character - 1` might neither be a valid index and nor be equal to the number of spaces before the first non-space character (in case there is a Unicode space character with > 1 codepoint?).